src/utils/mapping/field-mapping-utils.ts

```diff
@@ -316,10 +316,7 @@ export function getFieldMapping(
 export function getCustomFieldMapping(fieldId: string): IFormFieldMapping {
   const customFieldCertificateHandlebar =
     createCustomFieldHandlebarName(fieldId)
-  console.log(
-    'Custom field addded with handlebar: ',
-    customFieldCertificateHandlebar
-  )
```

Comment:
@rikukissa why did this get removed.  It was helpful to IET so countries could see the handlebar they needed for the certificate on a custom field.